docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+networks:
+  DB_net:
+    name: net1
+    driver: bridge
+services:
+  pythonapp:
+    command: /bin/bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"
+    container_name: todoapp-2.0.0
+    image: my-django-app:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        PYTHON_VERSION: 3.8
+    ports:
+      - "8080:8080"
+    networks:
+      - DB_net
+    depends_on:
+      - mysql
```

Comment:
The `depends_on` directive is used to specify the dependency order of services. However, it does not wait for the database to be ready before starting the `pythonapp`. Consider using a wait-for-it script or similar mechanism to ensure the database is ready before the application starts.